interfaces/seccomp/backend.go

```diff
@@ -81,6 +83,19 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	if err != nil {
 		return fmt.Errorf("cannot synchronize security files for snap %q: %s", snapName, err)
 	}
+
+	for baseName := range content {
+		in := filepath.Join(dirs.SnapSeccompDir, baseName)
+		out := filepath.Join(dirs.SnapSeccompDir, baseName+".bpf")
```

Comment:
My original approach was to use the same dir, however there are some complications. We use `EnsureDirState()` in the code that generates the seccomp profiles. This code will remove all of the old seccomp profiles (written by the old 2.24 code). So when we revert we run into the same problem that we need to solve, there will be new incompatible profiles at startup until 2.24 ran and generated valid profiles for 2.24. The easiest way out of this was to use a new dir. We can add a upcoming task to the forum to ensure we garbage collect the now-longer-used directory. We can of course still keep the `/var/lib/snapd/seccomp/profiles/` directory, when we do this, we need to tweak the profile names so that the glob used in `EnsureDirState()` does not remove the old 2.24 seccomp profiles. If that is more desirable than the new directory in can investigate this in my morning.